src/common/helper.ts

```diff
@@ -309,9 +309,14 @@ async function readProtocolStream(
   handle: string,
   path?: string
 ): Promise<Buffer> {
+  if (!isNode && path) {
+    throw new Error('Cannot write to a path outside of Node.js environment.');
+  }
+
```

Comment:
Can't we do `const fs = ...` here? (It seems fine to rely on Node.js' caching regardless, but if we can avoid it and simplify the code at the same time...)

Reply:
I meant `let fs` — updated comment to clarify

Reply:
👍 PTAL!